test/caching-precompiler.js

```diff
@@ -145,7 +146,7 @@ test('uses babelConfig for babel options when babelConfig is an object', functio
 	t.true('inputSourceMap' in options);
 	t.false(options.babelrc);
 	t.same(options.presets, ['stage-2', 'es2015']);
-	t.same(options.plugins, [customPlugin, powerAssert, rewrite, transformRuntime]);
+	t.same(options.plugins, [customPlugin, powerAssert, throwsHelper, rewrite, transformRuntime]);
```

Comment:
Sure, but `power-assert` might rewrite the `t.throws()` argument expression such that it is computed outside of `t.throws()`. Again, I don't know if it does, but if it did then you wouldn't be able to catch it.